tools/stats/upload_test_stats.py

```diff
@@ -123,12 +122,6 @@ def get_tests(workflow_run_id: int, workflow_run_attempt: int) -> List[Dict[str,
         for path in s3_paths:
             unzip(path)
 
-        artifact_paths = download_gha_artifacts(
```

Comment:
We probably have them all on S3 now with ODIC, but plz double check edge cases like ROCm

Reply:
I'm confused, I thought all the artifacts would eventually get uploaded to s3 by the step that uploads from gha artifacts to s3?

Reply:
Yup, that's my thought too.  Although the workflow needs to be added there explicitly https://github.com/pytorch/pytorch/blob/main/.github/workflows/upload-test-stats.yml#L5